src/bin/nydusd/api_server_glue.rs

```diff
@@ -326,6 +327,13 @@ impl ApiServer {
         }
     }
 
+    fn blob_cache_gc(&self, blob_id: String) -> ApiResponse {
+        self.get_daemon_object()?
```

Comment:
If we add `delete_blob()` method to `BlobCacheMgr`, then cand do
```
if let Some(blob_mgr) = DAEMON_CONTROLLER.get_blob_cache_mgr() {
     blob_mgr.delete_blob(blob_id).map_err(..).map(|_|...)
}
```

Reply:
Thanks , this should be better , will fix this.

Reply:
seems it is more more complicated to add delete_blob() to BlobCacheMgr , because we need to send  inuse and cull cmd via cachefiles devfd which is keeped in FsCacheHandler.